src/eth1spec/evm/gas.py

```diff
@@ -16,6 +16,9 @@
 from .error import OutOfGasError
 
 GAS_VERY_LOW = U256(3)
+GAS_STORAGE_SET = U256(20000)
+GAS_STORAGE_RESET = U256(5000)
```

Comment:
```suggestion
GAS_STORAGE_UPDATE = U256(5000)
```

I think this should be named different. IMO the connotation of "reset" is closer to "clear" than to literally "set again".

Reply:
I agree, but this was how it was named in the yellowpaper. Nevertheless, I will change it. Thanks.